llvm/lib/Target/AArch64/MCTargetDesc/AArch64InstPrinter.cpp

```diff
@@ -988,6 +988,22 @@ bool AArch64InstPrinter::printSysAlias(const MCInst *MI,
       Name = std::string(AT->Name);
     }
     break;
+    // Overlaps with AT and DC
```

Comment:
When is this used? I this used by the CMO to the outer cache level?

Reply:
Yeah. These are for the newly introduced DC instructions

Reply:
So why are we also testing AArch64AT::AT *AT = AArch64AT::lookupATByEncoding(Encoding); if this is only for DC instructions

Reply:
The newly added DC instructions map to the same case (15) as the AT instructions as they have overlapping encodings